src/panels/MinimalPanel.vue

```diff
@@ -59,11 +68,28 @@ export default {
   components: {
     retriever,
   },
+  data() {
+    return {
+      isHeaderAtBottom: false,
```

Comment:
This variable is minimal panel specific due to its design ( the header can shift to the bottom ), and is simply a 'copy' of `localExpanded` that is updated on certain animation events.
Its purpose is to update the header content ( the replacements of `localExpanded` with `isHeaderAtBottom` in the template above ) only once the animation has finished ( animation hooks below )